doc/getting-started/building-from-source.rst

```diff
@@ -79,8 +79,16 @@ Install the necessary tools for the following systems:
           libblkid-dev \
           e2fslibs-dev \
           pkg-config \
-          libnuma-dev
+          libnuma-dev \
+          liblz4-tool \
+          flex \
+          bison
+
      $ sudo pip3 install kconfiglib
+     $ wget https://acpica.org/sites/acpica/files/acpica-unix-20191018.tar.gz
```

Comment:
Another very important note is that this is **not** needed on the development machine. This is only really needed to be done in the Service VM. So this section is only relevant if you're using the Service VM as your dev machine too.

Reply:
> Another very important note is that this is **not** needed on the development machine. This is only really needed to be done in the Service VM. So this section is only relevant if you're using the Service VM as your dev machine too.

From V2.2, we use the iasl tool to compile offline ACPI binary for pre-launched VMs while building ACRN,
so we need to install the iasl tool in the ACRN building environment.

Reply:
> From V2.2, we use the iasl tool to compile offline ACPI binary for pre-launched VMs while building ACRN,
> so we need to install the iasl tool in the ACRN building environment.

This is good to know!